plugins/resolvectl/resolvectl

```diff
@@ -0,0 +1,121 @@
+#!/bin/sh
+# -*- sh -*-
+
+set -e
+
+: << =cut
+
+=head1 NAME
+
+resolvectl - Plugin to monitor resolvectl statistics.
+
+=head1 APPLICABLE SYSTEMS
+
+Any server running systemd-resolved
+
+=head1 CONFIGURATION
+
+[resolvectl]
+user root
+
+=head1 NOTES
+
+Uses the command "resolvectl statistics"
+
+=head1 AUTHOR
+
+Scott Hollenbeck <sah6284@gmail.com>
+
+=head1 LICENSE
+
+GPLv2
+
+=head1 VERSION
+
+1.0
+
+=head1 MAGIC MARKERS
+
+ #%# family=auto
+ #%# capabilities=autoconf
+
+=cut
+
+. "$MUNIN_LIBDIR/plugins/plugin.sh"
+
+if [ "$1" = "autoconf" ]; then
+	if ( resolvectl statistics >/dev/null 2>&1 ); then
+		echo yes
+		exit 0
+	else
+		if [ $? -eq 127 ]; then
+			echo "no (could not run \"resolvectl\")"
+			exit 0
+		else
+			echo "no (unknown resolvectl return value $?)"
```

Comment:
Minor problem here with reporting the return value of the `resolvctl` command: https://www.shellcheck.net/wiki/SC2319

Reply:
Thanks. I just fixed that code and committed the change.